core/src/main/java/org/mule/runtime/core/el/DefaultExpressionManager.java

```diff
@@ -262,9 +264,8 @@ public ValidationResult validate(String expression) {
     return success();
   }
 
-  protected static boolean isDwExpression(String expression) {
-    //TODO: MULE-10410 - Remove once DW is the default language.
-    return expression.startsWith(DEFAULT_EXPRESSION_PREFIX + DW_PREFIX) || expression.startsWith(DW_PREFIX);
+  private boolean hasMelExpression(String expression) {
```

Comment:
would it be valid for the 'mel' token to be in the middle of an expression instead of at the start?